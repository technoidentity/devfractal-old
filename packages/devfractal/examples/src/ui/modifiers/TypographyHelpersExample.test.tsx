import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Typography } from './TypographyHelpersExample'

test('Typography Helpers Example', () => {
  const component: ReactTestRenderer = renderer.create(<Typography />)
  expect(component.toJSON()).toMatchSnapshot()
})
