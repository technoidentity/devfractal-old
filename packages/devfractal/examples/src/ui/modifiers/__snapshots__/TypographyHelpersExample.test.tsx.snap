// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography Helpers Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Sizes
    </h1>
    <div
      className="box"
    >
      <div
        className="is-size-1"
      >
        size 1
      </div>
      <div
        className="is-size-2"
      >
        size 2
      </div>
      <div
        className="is-size-3"
      >
        size 3
      </div>
      <div
        className="is-size-4"
      >
        size 4
      </div>
      <div
        className="is-size-5"
      >
        size 5
      </div>
      <div
        className="is-size-6"
      >
        size 6
      </div>
      <div
        className="is-size-7"
      >
        size 7
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Alignment
    </h1>
    <div
      className="box"
    >
      <div
        className="has-text-centered"
      >
         centered
      </div>
      <div
        className="has-text-left"
      >
         left
      </div>
      <div
        className="has-text-right"
      >
         right
      </div>
      <div
        className="has-text-justified"
      >
        justified
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Responsive Size
    </h1>
    <div
      className="box"
    >
      <div
        className="is-size-1-mobile"
      >
        is-size-1-mobile
      </div>
      <div
        className="is-size-1-tablet"
      >
        is-size-1-tablet
      </div>
      <div
        className="is-size-1-touch"
      >
        is-size-1-touch
      </div>
      <div
        className="is-size-1-desktop"
      >
        is-size-1-desktop
      </div>
      <div
        className="is-size-1-widescreen"
      >
        is-size-1-widescreen
      </div>
      <div
        className="is-size-1-fullhd"
      >
        is-size-1-fullhd
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Responsive Alignment
    </h1>
    <div
      className="box"
    >
      <div
        className="has-text-centered-desktop has-text-desktop"
      >
        has-text-centered-desktop
      </div>
      <div
        className="has-text-left-mobile has-text-mobile"
      >
        has-text-left-mobile
      </div>
      <div
        className="has-text-right-tablet has-text-tablet"
      >
        has-text-right-tablet
      </div>
      <div
        className="has-text-centered-fullhd has-text-fullhd"
      >
        has-text-centered-fullhd
      </div>
      <div
        className="has-text-left-touch has-text-touch"
      >
        has-text-right-touch
      </div>
      <div
        className="has-text-right-widescreen has-text-widescreen"
      >
        has-text-right-widescreen
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Colors
    </h1>
    <div
      className="box"
    >
      <div
        className="has-text-black"
      >
        black
      </div>
      <div
        className="has-text-primary"
      >
        primary
      </div>
      <div
        className="has-text-success"
      >
        success
      </div>
      <div
        className="has-text-warning"
      >
        warning
      </div>
      <div
        className="has-text-link"
      >
        link
      </div>
      <div
        className="has-text-dark"
      >
        dark
      </div>
      <div
        className="has-text-info"
      >
        info
      </div>
      <div
        className="has-text-white"
      >
        white
      </div>
      <div
        className="has-text-danger"
      >
        Danger
      </div>
      <div
        className="has-text-black-bis"
      >
        black-bis
      </div>
      <div
        className="has-text-black-ter"
      >
        black-ter
      </div>
      <div
        className="has-text-grey-darker"
      >
        grey-darker
      </div>
      <div
        className="has-text-grey-dark"
      >
        grey-dark
      </div>
      <div
        className="has-text-grey"
      >
        grey
      </div>
      <div
        className="has-text-grey-light"
      >
        grey-light
      </div>
      <div
        className="has-text-grey-lighter"
      >
        grey-lighter
      </div>
      <div
        className="has-text-white-ter"
      >
        white-ter
      </div>
      <div
        className="has-text-white-bis"
      >
        White-bis
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Text Transformation
    </h1>
    <div
      className="box"
    >
      <div
        className="is-lowercase"
      >
        all characters to lowercase
      </div>
      <div
        className="is-uppercase"
      >
        all characters to uppercase
      </div>
      <div
        className="is-capitalized"
      >
        first character of each word to uppercase
      </div>
      <div
        className="is-italic"
      >
        all characters to italic
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Text Weight
    </h1>
    <div
      className="box"
    >
      <div
        className="has-text-weight-normal"
      >
        Normal
      </div>
      <div
        className="has-text-weight-light"
      >
        Light
      </div>
      <div
        className="has-text-weight-semiBold"
      >
        SemiBold
      </div>
      <div
        className="has-text-weight-bold"
      >
        Bold
      </div>
    </div>
  </section>,
]
`;
